models/core/sushiswap_lp_actions.sql

```diff
@@ -0,0 +1,127 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='log_id',
+        tags=['core', 'sushiswap_lp_actions'],
+        cluster_by=['log_id']
+        )
+}}
+
+
+with 
+
+events as (
+    select 
+        log_id,
+        block_timestamp,
+        date_trunc('day', block_timestamp) as block_date,

```

Comment:
no need for block_date

Reply:
Oh okay even if I'm using it to merge to `tokenprices`?

Reply:
oh okay, that's fine then..